plug.vim

```diff
@@ -935,7 +940,8 @@ function! s:log(bullet, name, lines)
     else
       let pos = 4
     endif
-    call append(pos - 1, s:format_message(a:bullet, a:name, a:lines))
+    let filtered = substitute(a:lines, '[[:cntrl:]]', '', 'g')
```

Comment:
So what are the `[[:cntrl:]]` characters we are filtering here? Is this an issue of Neovim? And if so, do you think we should do this as a safety net even when the issue is resolved?